web/src/components/Selector.tsx

```diff
@@ -1,62 +1,33 @@
-import { css } from '@emotion/react'
+import List from '@mui/material/List'
+import ListItem from '@mui/material/ListItem'
+import ListItemButton from '@mui/material/ListItemButton'
 import Tooltip from '@mui/material/Tooltip'
-import { styled, Theme } from '@mui/material/styles'
+import Typography from '@mui/material/Typography'
+import { styled } from '@mui/material/styles'
 import React, { ReactElement } from 'react'
 
 import SelectorItemModel from '../models/SelectorItemModel'
 import Link from './base/Link'
 
-const selectorItemStyle = ({ theme }: { theme: Theme }) => css`
-  height: 64px;
-  min-width: 90px;
-  padding: 0 5px;
-  font-size: 1.2em;
-  line-height: 64px;
-  text-align: center;
-  white-space: nowrap;
+const SelectorItemButton = styled(ListItemButton)<{ disabled?: boolean }>`
+  height: 48px;
+  min-width: 112px;
+  padding: 0 20px;
   border-radius: 30px;
-  transition:
-    background-color 0.2s,
-    border-radius 0.2s;
-  user-select: none;
-
-  ${theme.breakpoints.down('md')} {
-    width: 100%;
-    flex: 1 1 auto;
-    font-size: 1em;
-  }
-`
-
-const SelectorItem = styled(Link)<{ selected: boolean }>`
-  ${selectorItemStyle};
-  color: ${props => props.theme.legacy.colors.textColor};
-  ${props =>
-    props.selected
-      ? 'font-weight: 700;'
-      : `&:hover {
-          font-weight: 700;
-          border-radius: 0;
-        }`}
-`
-
-const DisabledSelectorItem = styled('div')`
-  ${selectorItemStyle};
-  color: ${props => props.theme.legacy.colors.textDisabledColor};
-`
-
-const BoldSpacer = styled('div')`
-  font-weight: 700;
-  height: 0;
-  overflow: hidden;
-  visibility: hidden;
-`
+  display: flex;
+  justify-content: center;
+  margin: 4px;
+  ${props => props.disabled && `color: ${props.theme.palette.text.disabled};`}
```

Comment:
🔧 This is already handled by mui, no need to set a custom text color.
```suggestion
  ${props => props.disabled && `color: ${props.theme.palette.text.disabled};`}
```